package.json

```diff
@@ -27,8 +27,9 @@
   "name": "@hint/monorepo",
   "private": true,
   "scripts": {
-    "build": "yarn clean && yarn build:hint && yarn build:utils && yarn build:connectors && npm-run-all --parallel build::parser:* && yarn build:utils-test && loop \"yarn build\" --cwd packages/ --exclude configuration-progressive-web-apps,configuration-web-recommended,connector-edge,hint,utils-debugging-protocol-common,utils-connector-tools,utils-tests-helpers,connector-chrome,connector-jsdom,parser-babel-config,parser-javascript,parser-manifest,parser-typescript-config,parser-webpack-config",
+    "build": "yarn clean && yarn build:hint && yarn build:utils && yarn build:connectors && npm-run-all --parallel build:parser:* && yarn build:utils-test && loop \"yarn build\" --cwd packages/ --exclude configuration-progressive-web-apps,configuration-web-recommended,connector-edge,hint,utils-debugging-protocol-common,utils-connector-tools,utils-tests-helpers,connector-chrome,connector-jsdom,parser-babel-config,parser-html,parser-javascript,parser-manifest,parser-typescript-config,parser-webpack-config",
```

Comment:
`parser:html` should be build before `connector-local`.

Reply:
Technically that should already happen as `connector-local` is built in `loop`, not as part of `build:connectors`.

Probably best to be explicit, though. I'll re-order these to run `build:parser:*` before `build:connectors` and add `connector-local` to the `build:connectors` step.

Sound good?

Reply:
> Probably best to be explicit

Yes, I think it's better to have it more explicit. Thanks!